docs/20_faq.md

```diff
@@ -5,25 +5,122 @@ sidebar_label: FAQ
 ---
 
 
-# FAQ
+# Frequently Asked Questions
 
 :::info
 
-Frequently asked questions about the ChainSafe gaming SDK.
+Frequently asked questions about ChainSafe Gaming's SDK, web3.unity.
 
 :::
 
-**Q**: I want to connect to other EVM compatible chains but it only lets me connect to Ethereum. How do I do that?
+## Common Questions
 
-**A**: You will need to edit the **network.js** file found within the WebTemplates folder. Edit the ChainId to the one for the blockchain you want to game to hit.
+### Is the SDK free to use?&#x20;
 
-**Q:**  I'm seeing the following error when I install the package. **The type or namespace name 'Newtonsoft' could not be found**
+Yes, the SDK is free to use.
 
-**A:** NewtonSoft is a dependency of the SDK and needs to be installed for the SDK to work. This could be found on the Unity Asset store [JSON.NET](https://assetstore.unity.com/packages/tools/input-management/json-net-for-unity-11347)
+### Can I make desktop games with the SDK?
 
-**Q:** I'm getting the following error when trying to import an NFT texture into the SDK. **Curl error 1: Protocol "ipfs" not supported or disabled in libcurl**
+Yes, you can make Desktop / Web and Mobile games using our SDK. You will need to create a Web3GL build to make desktop / mobile games with web3.unity.
 
-**A:** IPFS has its own protocol that isn't supported in JSON.Net in order to get around that you need to add something like the following code snippet.
+### I want to connect to other EVM-compatible chains but it only lets me connect to Ethereum. How do I do that?
```

Comment:
> but it only lets me connect to Ethereum

Reword to: "Does the SDK support other EVM-compatible chains besides Ethereum?

Also, just because we can visit Chainlist, does this imply that **ALL** of these EVM-compatible chains are supported inherently by the SDK, or is there only a specific list of supported chains?